src/Routers/ClassesRouter.js

```diff
@@ -57,14 +57,14 @@ export class ClassesRouter extends PromiseRouter {
       }
     }
 
-    if (typeof body.keys === 'string') {
```

Comment:
This is basically the real problem with `keys` and `excludeKeys`. If you look at `include` below, it takes an array of strings also and is not programmed in this way. `include` works fine for all of my tests